homeassistant/components/incomfort/__init__.py

```diff
@@ -37,7 +37,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: InComfortConfigEntry) ->
         raise NoHeaters from exc
     except IncomfortError as exc:
         if isinstance(exc.message, ClientResponseError):
```

Comment:
I think this is because the message is never of this type. Did you mean

```suggestion
         if isinstance(exc, ClientResponseError):
```